problemsets-mat223/modules/module8.tex

```diff
@@ -23,7 +23,30 @@
 	where $\vec b_1=\xhat+\yhat$, and $\vec b_2 =3\yhat$ be another basis for $\R^2$. Given that $\vec v=2\xhat-\yhat$, 
 	find $[\vec v]_{\mathcal E}$ and $[\vec v]_{\mathcal B}$.
 
-	XXX Finish
+	Given that $\vec v=2\xhat-\yhat$, we could find 
+	\[
+	    [\vec v]_{\mathcal E}=\mat{2\\-1}.
+	\]
+	To find $[\vec v]_{\mathcal B}$, we could use a system of equations to find $\vec v$ as a linear combination of $\vec b_1$ and $\vec b_2$. That is, set 
```

Comment:
`could` -> `can`.  "could" indicates a counterfactual. "We *could* do it this way, but we won't. We'll do it this other way." However, we are doing it this way, so you say "We *can* do it this way"

Reply:
Ohhhh I see. I thought it is modest